en/SUMMARY.md

```diff
@@ -33,7 +33,7 @@
 * [Basic Configuration](config/README.md)
   * [Firmware](config/firmware.md)
   * [Airframe](config/airframe.md)
-  * [Sensor Orientation](config/flight_controller_orientation.md)
+  * [Orientation](config/flight_controller_orientation.md)
```

Comment:
Revert this or justify please. I think sensor orientation accurate and helpful.

Reply:
Hum the problem was that it was dealing with sensor & flight controller orientation, but now that I think about it, flight controller's orientation is relevant because of hard mounted internal sensors inside :rofl: 

Good catch!